sql/parse_tree_nodes.h

```diff
@@ -3231,6 +3231,18 @@ class PT_create_table_default_collation : public PT_create_table_option {
   bool contextualize(Table_ddl_parse_context *pc) override;
 };
 
+class PT_ignore_create_table_option : public PT_create_table_option {
+  typedef PT_create_table_option super;
+
+ public:
+  explicit PT_ignore_create_table_option() {}
```

Comment:
No need to mark zero-arg constructor as explicit. Should it call parent default constructor, if any? Can it be marked noexcept while we are at that?

Reply:
I removed the constructor.